pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -434,24 +434,30 @@ def test_dti_tz_localize_utc_conversion(self, tz):
         with pytest.raises(pytz.NonExistentTimeError):
             rng.tz_localize(tz)
 
+    @pytest.mark.parametrize('tz', [
+        'UTC', 'US/Eastern', 'Europe/London', 'Asia/Tokyo', 'Australia/NSW',
+        'dateutil/US/Pacific', 'dateutil/Asia/Singapore', tzutc(),
+        pytz.FixedOffset(300), pytz.FixedOffset(0), pytz.FixedOffset(-300)
+    ])
     @pytest.mark.parametrize('idx', [
-        date_range(start='2014-01-01', end='2014-12-31', freq='M'),
-        date_range(start='2014-01-01', end='2014-12-31', freq='D'),
-        date_range(start='2014-01-01', end='2014-03-01', freq='H'),
-        date_range(start='2014-08-01', end='2014-10-31', freq='T')
+        date_range(start='2014-12-01', end='2015-02-28', freq='D'),
+        date_range(start='2014-05-01', end='2015-07-31', freq='D'),
+        date_range(start='2014-01-01', end='2014-02-28', freq='H'),
+        date_range(start='2014-08-01', end='2014-09-30', freq='15T')
```

Comment:
We can remove this parametrization and just use one `date_range` which should hopefully never have a DST transition.

Reply:
done